### PR TITLE
docs: Documentation enhancements

### PR DESCRIPTION
**Pull request by ItaloSa**
# What

This PR adds the missing part of the docs on how to get started using the theme. 

**Notice:** Due nature of the engineering feature that we're using and the updates that Docusaurus are receiving, we had breaking changes. Because of that, I'm recommending to the users use the same version of our example. We could work in future updates to be compatible with newer versions of the tool.

# How
Add to the `README.md` into the `Getting Started` section the steps on how to install the theme on Docusaurus.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,31 @@ This project was created to provide a consistent experience trought iFood projec
 - [Docusaurus](https://docusaurus.io/docs) >= 2.0.0
 - [Node](https://nodejs.org/en/download/) >= 16.14.0
 
-*tbd*
+1. Install docusaurus
```
Comment (adrcav) [inline README.md:18]: just a suggestion, this topic could be shown with two bash examples:

1. Install docusaurus 

> Due the unstable version of Docusaurus, we currently recomend use the version `2.0.0-beta.9`. We don't guarantee that the theme will work properly in other versions. 

```bash
npx create-docusaurus@2.0.0-beta.9 [project-name] [template]
```

Example:

```bash
npx create-docusaurus@2.0.0-beta.9 test-idt classic
```

what you think?

**Comment by evertontavares-ifood** [comment]
As a suggestion, I think that it`s good to have a template screenshot in the README file.

File: README.md (hunk 1 above)
Comment (ItaloSa) [reply README.md:18]: Nice! Just a point, the template must be the `classic` due the customization that we made

Comment (adrcav) [reply README.md:18]: that's a good point! this disclaimer should be described on this topic too.

**Comment by ItaloSa** [comment]
> Missing docs:
> 
> * [ ]  CONTRIBUTING
> * [ ]  LICENSE
> * [ ]  Project page - can be used as docusaurus example.
> * [ ]  Link the Code of Conduct inside README. (https://github.com/ifood/.github/blob/main/CODE_OF_CONDUCT.md)

The project page I think would be better be another issue

**Comment by evertontavares-ifood** [comment]
> 

OK!

**Review by evertontavares-ifood — APPROVED** [review]

**Pull request merged**